### DISCUSSION TITLE
Option to increase number displayed hits?

### DISCUSSION THREAD

**Discussion by yenalsekim** [community/community#50175] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Actual Question: 
Has the team considered adding the ability to increase the number of hits that come up on a single page?

Not a sarcastic and stuff. Purely acedemic: 
 Just curious if there are technical reasons. There are no advertisers so I'm assuming that's the case.
If the reason is technical could you very briefly describe it?

Non Complainy,
Yenals Ekim

**Comment by raghedawawdah** [comment]
What are your thoughts
